src/org/omegat/gui/dictionaries/DictionariesTextArea.java

```diff
@@ -240,11 +242,10 @@ protected void setFoundResult(final SourceTextEntry se, final List<DictionaryEnt
         if (!data.isEmpty() && Preferences.isPreference(Preferences.NOTIFY_DICTIONARY_HITS)) {
             scrollPane.notify(true);
         }
-
-        StringBuilder txt = new StringBuilder();
         boolean wasPrev = false;
-        int i = 0;
-        for (DictionaryEntry de : data) {
+        StringBuilder txt = new StringBuilder();
+        for (int i = 0, size = data.size(); i < size; i++) {
```

Comment:
You changed the loop to a manually counting loop, but you left the existing increment to `i` so you actually skip some dictionary entries now.

Reply:
I have not fixed yet.

Reply:
thanks.